client/src/elm/Translate.elm

```diff
@@ -15303,202 +15303,202 @@ translationSet trans =
+            { english = "Hello " ++ name ++ " Welcome to the work-based resilience program. As healthcare providers, we often face obstacles and challenges; for example: a lot of work, various responsibilities (including those at home), listening to patients with different problems, etc."
+            , kinyarwanda = Just <| "Muraho neza " ++ name ++ ", Murakaza neza muri gahunda yo kumenya kwiyitaho bitewe n'akazi dukora. Nk'abakozi dutanga serivisi z'ubuvuzi, dukunze guhura n'imbogamizi; urugero: akazi kenshi, inshingano nyinshi harimo n'izo mu rugo, gutega amatwi abantu bafite ibibazo bitandukanye baba baje batugana, n'ibindi."
             }
 
         ResilienceMessageIntroduction1Paragraph2 ->
             { english = "You will be receiving short messages 3 times a week for six months. Those messages will help healthcare workers like us make progress in the quality of our work and learn how to cope with extreme fatigue and stress."
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Muzajya mwakira ubutumwa bugufi (messages) inshuro 3 mu cyumweru, mu gihe cy'amezi atandatu. Ubwo butumwa buzafasha abakozi batanga serivisi z'ubuvuzi nkatwe, gutera intambwe nziza mu kumenya uko dukora akazi kacu neza, no guhangana n'umunaniro ukabije."
             }
 
         ResilienceMessageIntroduction1Paragraph3 ->
-            { english = "The aim of these messages is to help you as a healthcare worker feel better and more confident in your work and your daily life"
-            , kinyarwanda = Nothing
+            { english = "The aim of these messages is to help you as a healthcare worker feel better and more confident in your work and your daily life."
+            , kinyarwanda = Just "Intego y'ubu butumwa ni ukugufasha nk'umukozi utanga serivisi z'ubuvuzi kumva umeze neza kandi wifitemo icyizere n'umutekano mu kazi ukora ndetse n'ubuzima bwawe bwa buri munsi."
             }
 
         ResilienceMessageIntroduction2Title ->
             { english = "More information about the program"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Andi makuru ku byerekeye iyi gahunda"
             }
 
         ResilienceMessageIntroduction2Paragraph1 ->
             { english = "There are several things we face at work such as being given too many responsibilities and working overtime. However, we can at least learn how to deal with extreme fatigue and stress."
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Hari ibintu byinshi duhura nabyo mu kazi nko guhabwa inshingano nyinshi, gukora amasaha y'ikirenga,... Ariko byibuze dushobora kwiga uko twarwanya umunaniro ukabije."
             }
 
         ResilienceMessageIntroduction2Paragraph2 ->
             { english = "After this six-month journey, we hope that you will be able to"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Nyuma y'uru rugendo rw'amezi atandatu, twizera ko uzaba ushobora:"
```

Comment:
Why `:` in the end? If it's needed, it should appear in English as well.

Reply:
@Diama1 ^^ ?

Reply:
it has been fixed 
![Screenshot from 2023-06-01 10-15-56](https://github.com/TIP-Global-Health/eheza-app/assets/32199791/b6ed2ecd-88ba-45a9-b10b-810284bbf36f)